_codux/boards/services-section.board.module.scss

```diff
@@ -0,0 +1,88 @@
+.servicesSection {
```

Comment:
If I use this snippet for anything other than a list of services (e.g. a features section, testimonials section), then these class names will not make sense.

So if you have a better idea how to name this layout, let's do that.

Also I see that you've used both "namespaced" names and "non-namespaced names".
Namespaced: "servicesSection", "servicesTitle", "servicesDescription", "serviceCardTitle"
Non-namespaced: "card", "content", "icon".

Both have pros and cons, but we should pick one convention and be consistent.